app/components/elements/breadcrumb_nav_component.html.erb

```diff
@@ -1,3 +1,4 @@
+<% content_for(:title, page_title_from_breadcrumbs) # set the page's html title from the breadcrumb nav %>
```

Comment:
Note: the application layout is already setup to render the `:title` when it is set here.  See https://github.com/sul-dlss/hungry-hungry-hippo/blob/main/app/views/layouts/application.html.erb#L13